### PR TITLE
Add updating guide

### PR DESCRIPTION
**Pull request by ylt**

### CODE REVIEW

**Comment by ylt** [comment]
Corresponding PHP guide: https://github.com/openactive/models-php/pull/47

**Review by henryaddison — COMMENTED** [review]
Running the script fails. Here's the final lines of output:

```
Working out known parents
Calculating inheritance trees
Picking the primary parents
TypeError: Cannot read property 'writeFile' of undefined
    at Ruby.writeFile (/Users/henryaddison/Simpleweb/oa/code/models-ruby/vendor/models-lib/src/generator.js:48:14)
    at tryCatch (/Users/henryaddison/Simpleweb/oa/code/models-ruby/vendor/models-lib/node_modules/@babel/node/node_modules/regenerator-runtime/runtime.js:45:40)
    at Generator.invoke [as _invoke] (/Users/henryaddison/Simpleweb/oa/code/models-ruby/vendor/models-lib/node_modules/@babel/node/node_modules/regenerator-runtime/runtime.js:271:22)
    at Generator.prototype.(anonymous function) [as next] (/Users/henryaddison/Simpleweb/oa/code/models-ruby/vendor/models-lib/node_modules/@babel/node/node_modules/regenerator-runtime/runtime.js:97:21)
    at asyncGeneratorStep (/Users/henryaddison/Simpleweb/oa/code/models-ruby/vendor/models-lib/src/generator.js:42:103)
    at _next (/Users/henryaddison/Simpleweb/oa/code/models-ruby/vendor/models-lib/src/generator.js:44:194)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
Bundler could not find compatible versions for gem "typesafe_enum":
  In Gemfile:
    openactive was resolved to 0.1.2, which depends on
      typesafe_enum

Could not find gem 'typesafe_enum', which is required by gem 'openactive', in any of the sources.
Bundler could not find compatible versions for gem "typesafe_enum":
  In Gemfile:
    openactive was resolved to 0.1.2, which depends on
      typesafe_enum

Could not find gem 'typesafe_enum', which is required by gem 'openactive', in any of the sources.
```
I'm using Node 8.93 and Ruby 2.6.6.

File: update-models.sh [hunk 1]
```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
```
Comment (henryaddison) [inline update-models.sh:7]: same as the PHP branch, I'd recommend adding something like `set -euo pipefail` so that the script exits on an error

Comment (ylt) [reply update-models.sh:7]: Done.

**Comment by ylt** [comment]
> ```
> Working out known parents
> Calculating inheritance trees
> Picking the primary parents
> TypeError: Cannot read property 'writeFile' of undefined
>     at Ruby.writeFile (/Users/henryaddison/Simpleweb/oa/code/models-ruby/vendor/models-lib/src/generator.js:48:14)
>     at tryCatch (/Users/henryaddison/Simpleweb/oa/code/models-ruby/vendor/models-lib/node_modules/@babel/node/node_modules/regenerator-runtime/runtime.js:45:40)
>     at Generator.invoke [as _invoke] (/Users/henryaddison/Simpleweb/oa/code/models-ruby/vendor/models-lib/node_modules/@babel/node/node_modules/regenerator-runtime/runtime.js:271:22)
>     at Generator.prototype.(anonymous function) [as next] (/Users/henryaddison/Simpleweb/oa/code/models-ruby/vendor/models-lib/node_modules/@babel/node/node_modules/regenerator-runtime/runtime.js:97:21)
>     at asyncGeneratorStep (/Users/henryaddison/Simpleweb/oa/code/models-ruby/vendor/models-lib/src/generator.js:42:103)
>     at _next (/Users/henryaddison/Simpleweb/oa/code/models-ruby/vendor/models-lib/src/generator.js:44:194)
>     at <anonymous>
>     at process._tickCallback (internal/process/next_tick.js:188:7)
> 
> I'm using Node 8.93 and Ruby 2.6.6.

This was due to nvm not being available in the context of the shell script, and the install being done on the wrong version. I've added in a node version check which should resolve this.

**Review by henryaddison — COMMENTED** [review]
I've updated the Rubocop config to deal with some errors due to changed names.

This looks good. Two small suggestions:
* Add a quick line in README pointing at new UPDATING.md document so the casual contributor knows it exists
* Add vendor (or vendor/models-lib) to gitignore so it doesn't get checked in by accident by another contributor

NB the tests are not passing as the models have changed again but the updating documentation looks good.

**Review by henryaddison — APPROVED** [review]
Looks good.

**Pull request merged**